src/Layout/redist/targets/RuntimeAnalyzers.targets

```diff
@@ -0,0 +1,24 @@
+<Project>
+
+  <PropertyGroup>
+    <RuntimeAnalyzersSourceRoot>$(ArtifactsBinDir)redist\$(Configuration)\dotnet\</RuntimeAnalyzersSourceRoot>
+    <NetCoreRuntimeAnalyzersSubPath>packs\Microsoft.NetCore.App.Ref\*\analyzers</NetCoreRuntimeAnalyzersSubPath>
```

Comment:
Will the SDK ever carry more than one pack?  I guess if it did, we'd still be OK to include multiple versions since you match based on the major and minor. 

How would this interact with VS on installation -- is it possible we'd ever have two of these analyzer packs installed, or would the newer one replace the old one?

Reply:
That should work fine - the analyzers are deployed with the version number in their path like

> C:\Program Files\Microsoft Visual Studio\2022\Preview\DotNetRuntimeAnalyzers\AspNetCoreAnalyzers\9.0.0-preview.7.24406.2\analyzers\dotnet\cs\Microsoft.AspNetCore.App.Analyzers.dll

so multiple of them could live side by side.

Reply:
Ok, so if that happens would ever have more than one of these DotNetRuntimeAnalyzers installer packages inserted into VS that are trying to install the same files?  I'm guessing no - if possible we should ensure that's the case.